.changeset/breezy-candies-joke.md

```diff
@@ -1,8 +1,7 @@
 ---
-"@fluid-experimental/devtools-core": minor
```

Comment:
I wonder what lead the original dev to get it wrong?  I guess they probably changed this package too so it was auto-suggested?  I'd caution against doing too much manual tweaking, it doesn't have to be perfect and is not sustainable IMO (you'd have a better feel of the burden - my impression is you're trying to find the balance and then hopefully we can continue to improve automation / training to yield the right result "automatically")

Reply:
> I wonder what lead the original dev to get it wrong?

They didn't. The change required updating devtools-core's tree2 usage. The "real change" was only in the Tree package. IMO this is a similar situation to examples - they often need to be updated when an API changes, but that API change is often noise in the changelog for individual packages.